### DISCUSSION TITLE
Asset management

### DISCUSSION THREAD

**Discussion by xibman** [sofn-xyz/mailing#225] [Q&A]
Hi and thanks a lot for this great project ! 

Something really interesting could be the assets management.

Like the email folder, have an other folder for the assets like images.

**Comment by alexfarrill** [comment]
Thanks for the idea @xibman !

**Reply by xibman** [reply]
@alexfarrill

To be really interesting  the link with image can be dynamic so we can use local image on dev and remote if we send an email.
(symlink between a folder and public folder from nextJs)

<MjmlImage src="`${image_variable}/images/ltest.png`" />

In dev image_variable = public folder
in prod / sending email = url to S3 folder or anything

**Reply by xibman** [reply]
@alexfarrill i partially solved the problem with a next env variable and a script that copy. my assets to the next public folder.

Simple but it work really well

**Reply by alexfarrill** [reply]
Awesome, let me know if there's a link I can check out, I'd love to see how it works for you.  In Rails, I've done a similar thing where it compiles assets on deploy to a folder that can then be served behind a CDN and cached for a long time

**Reply by xibman** [reply]
Hi @alexfarrill  i just share with you a repository where you can see how i achieve the asset linkage.

Just a simple script that copy my assets folder to the one in .mailing/public folder 
And an env variable used by next

**Reply by alexfarrill** [reply]
Oops, I didn't get around to this before the link expired, would you mind sending me another invite @xibman ?

**Reply by xibman** [reply]
@alexfarrill  Done

**Reply by alexfarrill** [reply]
✔️ thanks!
